src/connectivities.jl

```diff
@@ -26,22 +26,22 @@ facetype(c::Connectivity) = facetype(typeof(c))
     connect(list, F)
 
 Connect a `list` of indices from a global vector of [`Point`](@ref)
-into a [`Face`](@ref) of type `F`.
+into a [`Polytope`](@ref) of type `F`.
 
 ## Example
 ```
 Δ = connect((1,2,3), Triangle)
 ```
 """
-connect(list::NTuple{N,Int}, F::Type{<:Face}) where {N} = Connectivity{F,N}(list)
+connect(list::NTuple{N,Int}, F::Type{<:Polytope}) where {N} = Connectivity{F,N}(list)
```

Comment:
Although I like all your suggestions so far, it would be nice if you could draft PRs more carefully. I am spotting a lot occurrences of incomplete changes like this where you chage Face to Polytope and forget to update the type parameter from F to P.